code/datums/mood_events/drug_events.dm

```diff
@@ -78,3 +78,8 @@
 	description = "<span class='nicegreen'>Eh ah AAAAH! HA HA HA HA HAA! Uuuh.</span>\n"
 	mood_change = 6
 	timeout = 3 MINUTES
+
+/datum/mood_event/antidepressant
+	description = "<span class='nicegreen'>All the bad things feel a bit more far away.</span>\n"
+	mood_change = 15

```

Comment:
this is not even an attempt at a balanced moodlet. This is basically just as good as happiness (as it can cancel out most moodlet situations, whether this is done with huge number A or huge number B doesn't really matter much.) without any of the side effects.